### PR TITLE
Add client-go 10.0.0 changelog

### PR DESCRIPTION
**Pull request by nikhita**
For https://github.com/kubernetes/client-go/issues/506

Ref: https://docs.google.com/spreadsheets/d/1DS9hI2u1wYMG70_ALqc4holhmovNz-fr-ZUQGvhoplM

/assign @sttts @caesarxuchao

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+* `GetScale` and `UpdateScale` methods have been added for `apps/v1` clients
+and with this, no-verb scale clients have been removed.
+([#70437](https://github.com/kubernetes/kubernetes/pull/70437))
+
+* `k8s.io/client-go/util/cert/triple` package has been removed.
+([#70966](https://github.com/kubernetes/kubernetes/pull/70966))
+
+**New Features:**
+
+* `--address` flag is added to the port-forward command to allow listening on
```
Comment (sttts) [inline CHANGELOG.md]: is this client-go?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+* `unfinished_work_microseconds` is added to the workqueue metrics.
+It can be used to detect stuck worker threads (kube-controller-manager runs many workqueues.).
+([#70884](https://github.com/kubernetes/kubernetes/pull/70884))
+
+* A method `GetPorts` is added to expose the ports that were forwarded.
+This can be used to retrieve the locally-bound port in cases where the input was port 0.
+([#67513](https://github.com/kubernetes/kubernetes/pull/67513))
+
+* Dynamic listers and informers, that work with `runtime.Unstructured` objects,
+are added. These are useful for wriitng generic, non-generated controllers.
```
Comment (sttts) [inline CHANGELOG.md]: typo

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+
+* The dynamic fake client now supports JSONPatch.
+([#69330](https://github.com/kubernetes/kubernetes/pull/69330))
+
+* The `GetBinding` method is added for pods in the fake client.
+([#69412](https://github.com/kubernetes/kubernetes/pull/69412))
+
+**Bug fixes and Improvements:**
+
+* The apiVersion and action name values for fake evictions is now set.
```
Comment (sttts) [inline CHANGELOG.md]: `apiVersion` in quotes

Comment (sttts) [reply CHANGELOG.md]: are now set

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+([#69035](https://github.com/kubernetes/kubernetes/pull/69035))
+
+* PEM files containing both TLS certificate and key can now be parsed in
+arbitrary order. Previously key was always required to be first.
+([#69536](https://github.com/kubernetes/kubernetes/pull/69536))
+
+* When `--rotate-server-certificates` is enabled, a new certificate is no longer
```
Comment (sttts) [inline CHANGELOG.md]: which component is this?

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+requested on startup if the current certificate on disk is satisfactory.
+([#69991](https://github.com/kubernetes/kubernetes/pull/69991))
+
+* Go clients created from a kubeconfig that specifies a `TokenFile` now
+periodically reload the token from the specified file.
+([#70606](https://github.com/kubernetes/kubernetes/pull/70606))
+
+* kube-controller-manager is reported as unhealthy if leader election is wedged
```
Comment (sttts) [inline CHANGELOG.md]: client-go?

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,129 @@ https://github.com/kubernetes/test-infra/issues/5843.
...
+* The default value of extensions/v1beta1 Deployment's `RevisionHistoryLimit`
+is set to `MaxInt32`. ([#66605](https://github.com/kubernetes/kubernetes/pull/66605))
+
+* `procMount` field is no longer incorrectly marked as required in openapi schema.
+([#69694](https://github.com/kubernetes/kubernetes/pull/69694))
+
+* The caBundle and service fields in admission webhook API objects now correctly
+indicate they are optional. ([#70138](https://github.com/kubernetes/kubernetes/pull/70138))
+
+* `kubectl apply` can now change a deployment strategy from rollout to recreate
```
Comment (sttts) [inline CHANGELOG.md]: client-go?

**Comment by nikhita** [comment]
Updated. Ptal.

**Comment by nikhita** [comment]
Ooops, 1.13 hasn't been released yet. :)

**Pull request merged**